tests/main_self_interaction.jl

```diff
@@ -11,49 +11,56 @@ include("src/constants.jl")
 function main()
     N = 4 # number of sites (NEED TO GO TILL 96 for Rog_results)
     cutoff = 1E-14 # specifies a truncation threshold for the SVD in MPS representation (SMALL CUTOFF = MORE ENTANGLEMENT)
-    tau = 0.05 # time step (NEED TO BE 0.05 for Rog_results)
+    τ = 0.05 # time step (NEED TO BE 0.05 for Rog_results)
     ttotal = 5 # total time of evolution (NEED TO GO TILL 50 for Rog_results)
-    tolerance  = 5E-1 # acceptable level of error or deviation from an exact value or solution
-    del_x = 1E-5 # length of the box of interacting neutrinos at a site/shape function width of neutrinos (cm)
+    tolerance  = 5E-1 # acceptable level of error or deviation from the exact value or solution
+    ∇x = 1E-3 # length of the box of interacting neutrinos at a site/shape function width of neutrinos in cm 
 
     # s is an array of spin 1/2 tensor indices (Index objects) which will be the site or physical indices of the MPS.
     # conserve_qns=true conserves the total spin quantum number "S" in the system as it evolves
     s = siteinds("S=1/2", N; conserve_qns=true)  
 
-    # Constants for Rogerro's fit (only interaction term)
+    # Constants for Rogerro's fit (only self-interaction term)
     a_t = 0
     b_t = 2.105
     c_t = 0
     
     # Initialize an array of ones for all N sites
     mu = ones(N) # erg
     
-    # n is the total number of neutrinos contained at each site (dimensionless)
-    n = mu .* fill((del_x)^3/(sqrt(2) * G_F), N)
+    # Create an array of dimension N and fill it with the value 1/(sqrt(2) * G_F). This is the number of neutrinos 
+    n = mu .* fill((∇x)^3/(sqrt(2) * G_F), N)
```

Comment:
I think a delta would be a better symbol than a nabla. This makes it look like a gradient of x.